Dockerfile.test

```diff
@@ -23,6 +23,6 @@ RUN pip3 install -r requirements.txt
 
 COPY . $INSTALL_DIR
 
-COPY ./test/configure_dbus.sh /configure_dbus.sh
+COPY ./test/generate_tests_exec_script.sh /generate_tests_exec_script.sh
 
-ENTRYPOINT ["./test/configure_dbus.sh"]
+ENTRYPOINT ["./test/generate_tests_exec_script.sh"]
```

Comment:
I dont think we need to say `_script` at the end of the naming because it is clear it is a script.

Reply:
what about just `test_executor.sh`

Reply:
`test_executor.sh` is the name of the file being generated.

Reply:
I know but the file being generated should be configure_dbus because that is what that generated file is essentially doing

Reply:
it has all the logic specific to dbus and nothing about the actual tests so my suggestion here is that the actual script that triggers the tests should be `test_executor.sh` and the dbus generated file should be `configure_dbus.sh`